metadata-ingestion/src/datahub/emitter/mcp.py

```diff
@@ -220,4 +222,8 @@ def as_workunit(self) -> "MetadataWorkUnit":
                 id=f"{self.entityUrn}-{self.aspectName}-{ts}", mcp=self
             )
```

Comment:
also need to preserve treat_errors_as_warnings here

Reply:
Nice catch